src/services/index.js

```diff
@@ -50,6 +50,54 @@ export const getFirstTag = (document: Document, localName: LocalName) => {
   return null;
 };
 
+export const createStyleProp = (
+  element: Element,
+  stylesheets: StyleSheets,
+  options: HvComponentOptions,
+) => {
+  const styleAttr = options.styleAttr || 'style';
+  if (!element.getAttribute(styleAttr)) {
+    return [];
+  }
+
+  const styleValue = element.getAttribute(styleAttr) || '';
+  const styleIds = styleValue.split(' ');
+  let styleRules = styleIds.map(s => stylesheets.regular[s]);
+
+  if (options.pressed) {
+    let pressedRules = styleIds
+      .map(s => stylesheets.pressed[s])
+      .filter(r => !!r);
```

Comment:
We can simplify this for filtering truthy values:

```suggestion
      .filter(Boolean);
```